kubectl-rook-ceph.sh

```diff
@@ -331,11 +417,14 @@ function run_main_command() {
   rook)
     rook_version "$@"
     ;;
+  health)
+    cluster_health "$@"
```

Comment:
When I wrote the skeleton, I intended functions that are running commands to start with `run_`. You don't have to follow this convention, but it can help visually identify core functions versus helpers as the file grows large.

```suggestion
    run_cluster_health "$@"
```